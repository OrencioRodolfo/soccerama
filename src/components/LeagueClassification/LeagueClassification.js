import React, { Component } from 'react'
import { connect } from 'react-redux'
import PropTypes from 'prop-types'
import {
  Table,
  TableBody,
  TableHeader,
  TableHeaderColumn,
  TableRow,
  TableRowColumn,
} from 'material-ui/Table'
import { getStandings } from '../../store/actions/standings'

class TableExampleSimple extends Component {
  static propTypes = {
    getStandings: PropTypes.func.isRequired,
    standings: PropTypes.arrayOf(PropTypes.shape({
      team_id: PropTypes.number.isRequired,
      team_name: PropTypes.string.isRequired,
      position: PropTypes.number.isRequired,
      overall: PropTypes.shape({
        draw: PropTypes.number.isRequired,
        games_played: PropTypes.number.isRequired,
        goals_scored: PropTypes.number.isRequired,
        lost: PropTypes.number.isRequired,
        won: PropTypes.number.isRequired,
      }).isRequired,
      total: PropTypes.shape({
        goal_difference: PropTypes.string.isRequired,
      }).isRequired,
      points: PropTypes.number.isRequired,
    })).isRequired,
  }

  componentDidMount() {
    this.props.getStandings(825)
  }

  renderStandingList() {
    return this.props.standings.map((item, index) => (
      <TableRow key={item.team_id}>
        <TableRowColumn>{index + 1}</TableRowColumn>
        <TableRowColumn>{item.team_name}</TableRowColumn>
        <TableRowColumn>{item.overall.games_played}</TableRowColumn>
        <TableRowColumn>{item.overall.won}</TableRowColumn>
        <TableRowColumn>{item.overall.draw}</TableRowColumn>
        <TableRowColumn>{item.overall.lost}</TableRowColumn>
        <TableRowColumn>{item.overall.goals_scored}</TableRowColumn>
        <TableRowColumn>{item.total.goal_difference}</TableRowColumn>
        <TableRowColumn>{item.points}</TableRowColumn>
      </TableRow>
    ))
  }

  render() {
    return (
      <Table>
        <TableHeader
          displaySelectAll={false}
          adjustForCheckbox={false}
        >
          <TableRow>
            <TableHeaderColumn>Position</TableHeaderColumn>
            <TableHeaderColumn>Team name</TableHeaderColumn>
            <TableHeaderColumn>Played</TableHeaderColumn>
            <TableHeaderColumn>Won</TableHeaderColumn>
            <TableHeaderColumn>Drawn</TableHeaderColumn>
            <TableHeaderColumn>Lost</TableHeaderColumn>
            <TableHeaderColumn>Goal</TableHeaderColumn>
            <TableHeaderColumn>Difference</TableHeaderColumn>
            <TableHeaderColumn>Points</TableHeaderColumn>
          </TableRow>
        </TableHeader>
        <TableBody displayRowCheckbox={false}>
          {this.renderStandingList()}
        </TableBody>
      </Table>
    )
  }
}

const mapStateToProps = ({ standings }) => ({
  standings: standings.standings,
})

export default connect(mapStateToProps, {
  getStandings,
})(TableExampleSimple)
export { TableExampleSimple as DTableExampleSimple }
