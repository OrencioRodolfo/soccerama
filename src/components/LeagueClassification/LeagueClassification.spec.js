import React from 'react'
import { shallow } from 'enzyme'
import { DTableExampleSimple } from './LeagueClassification'

describe('LeagueClassification component', () => {
  let getStandings
  let standings
  beforeEach(() => {
    getStandings = jest.fn()
    standings = [
      {
        team_id: 1,
        team_name: 'sample',
        position: 2,
        overall: {
          draw: 3,
          games_played: 4,
          goals_scored: 5,
          lost: 6,
          won: 7,
        },
        total: {
          goal_difference: '+1',
        },
        points: 8,
      },
    ]
  })

  it('must render properly', () => {
    const component = shallow(<DTableExampleSimple
      standings={standings}
      getStandings={getStandings}
    />)

    expect(component).toMatchSnapshot()
  })
})
