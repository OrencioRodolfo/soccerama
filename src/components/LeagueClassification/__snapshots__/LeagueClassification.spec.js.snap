// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeagueClassification component must render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TableExampleSimple
    getStandings={
      [MockFunction] {
        "calls": Array [
          Array [
            825,
          ],
        ],
      }
    }
    standings={
      Array [
        Object {
          "overall": Object {
            "draw": 3,
            "games_played": 4,
            "goals_scored": 5,
            "lost": 6,
            "won": 7,
          },
          "points": 8,
          "position": 2,
          "team_id": 1,
          "team_name": "sample",
          "total": Object {
            "goal_difference": "+1",
          },
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "allRowsSelected": false,
      "children": Array [
        <TableHeader
          adjustForCheckbox={false}
          displaySelectAll={false}
          enableSelectAll={true}
          selectAllSelected={false}
        >
          <TableRow
            displayBorder={true}
            hoverable={false}
            hovered={false}
            selectable={true}
            striped={false}
          >
            <TableHeaderColumn>
              Position
            </TableHeaderColumn>
            <TableHeaderColumn>
              Team name
            </TableHeaderColumn>
            <TableHeaderColumn>
              Played
            </TableHeaderColumn>
            <TableHeaderColumn>
              Won
            </TableHeaderColumn>
            <TableHeaderColumn>
              Drawn
            </TableHeaderColumn>
            <TableHeaderColumn>
              Lost
            </TableHeaderColumn>
            <TableHeaderColumn>
              Goal
            </TableHeaderColumn>
            <TableHeaderColumn>
              Difference
            </TableHeaderColumn>
            <TableHeaderColumn>
              Points
            </TableHeaderColumn>
          </TableRow>
        </TableHeader>,
        <TableBody
          allRowsSelected={false}
          deselectOnClickaway={true}
          displayRowCheckbox={false}
          multiSelectable={false}
          preScanRows={true}
          selectable={true}
          style={Object {}}
        >
          <TableRow
            displayBorder={true}
            hoverable={false}
            hovered={false}
            selectable={true}
            striped={false}
          >
            <TableRowColumn
              hoverable={false}
            >
              1
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              sample
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              4
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              7
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              3
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              6
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              5
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              +1
            </TableRowColumn>
            <TableRowColumn
              hoverable={false}
            >
              8
            </TableRowColumn>
          </TableRow>
        </TableBody>,
      ],
      "fixedFooter": true,
      "fixedHeader": true,
      "height": "inherit",
      "multiSelectable": false,
      "selectable": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "adjustForCheckbox": false,
          "children": <TableRow
            displayBorder={true}
            hoverable={false}
            hovered={false}
            selectable={true}
            striped={false}
          >
            <TableHeaderColumn>
              Position
            </TableHeaderColumn>
            <TableHeaderColumn>
              Team name
            </TableHeaderColumn>
            <TableHeaderColumn>
              Played
            </TableHeaderColumn>
            <TableHeaderColumn>
              Won
            </TableHeaderColumn>
            <TableHeaderColumn>
              Drawn
            </TableHeaderColumn>
            <TableHeaderColumn>
              Lost
            </TableHeaderColumn>
            <TableHeaderColumn>
              Goal
            </TableHeaderColumn>
            <TableHeaderColumn>
              Difference
            </TableHeaderColumn>
            <TableHeaderColumn>
              Points
            </TableHeaderColumn>
          </TableRow>,
          "displaySelectAll": false,
          "enableSelectAll": true,
          "selectAllSelected": false,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <TableHeaderColumn>
                Position
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Team name
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Played
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Won
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Drawn
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Lost
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Goal
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Difference
              </TableHeaderColumn>,
              <TableHeaderColumn>
                Points
              </TableHeaderColumn>,
            ],
            "displayBorder": true,
            "hoverable": false,
            "hovered": false,
            "selectable": true,
            "striped": false,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Position",
              },
              "ref": null,
              "rendered": "Position",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Team name",
              },
              "ref": null,
              "rendered": "Team name",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Played",
              },
              "ref": null,
              "rendered": "Played",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Won",
              },
              "ref": null,
              "rendered": "Won",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Drawn",
              },
              "ref": null,
              "rendered": "Drawn",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Lost",
              },
              "ref": null,
              "rendered": "Lost",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Goal",
              },
              "ref": null,
              "rendered": "Goal",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Difference",
              },
              "ref": null,
              "rendered": "Difference",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "Points",
              },
              "ref": null,
              "rendered": "Points",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "allRowsSelected": false,
          "children": Array [
            <TableRow
              displayBorder={true}
              hoverable={false}
              hovered={false}
              selectable={true}
              striped={false}
            >
              <TableRowColumn
                hoverable={false}
              >
                1
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                sample
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                4
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                7
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                3
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                6
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                5
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                +1
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                8
              </TableRowColumn>
            </TableRow>,
          ],
          "deselectOnClickaway": true,
          "displayRowCheckbox": false,
          "multiSelectable": false,
          "preScanRows": true,
          "selectable": true,
          "style": Object {},
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "1",
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <TableRowColumn
                  hoverable={false}
                >
                  1
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  sample
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  4
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  7
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  3
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  6
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  5
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  +1
                </TableRowColumn>,
                <TableRowColumn
                  hoverable={false}
                >
                  8
                </TableRowColumn>,
              ],
              "displayBorder": true,
              "hoverable": false,
              "hovered": false,
              "selectable": true,
              "striped": false,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 1,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 1,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "sample",
                  "hoverable": false,
                },
                "ref": null,
                "rendered": "sample",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 4,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 4,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 7,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 7,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 3,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 3,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 6,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 6,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 5,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 5,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "+1",
                  "hoverable": false,
                },
                "ref": null,
                "rendered": "+1",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": 8,
                  "hoverable": false,
                },
                "ref": null,
                "rendered": 8,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "allRowsSelected": false,
        "children": Array [
          <TableHeader
            adjustForCheckbox={false}
            displaySelectAll={false}
            enableSelectAll={true}
            selectAllSelected={false}
          >
            <TableRow
              displayBorder={true}
              hoverable={false}
              hovered={false}
              selectable={true}
              striped={false}
            >
              <TableHeaderColumn>
                Position
              </TableHeaderColumn>
              <TableHeaderColumn>
                Team name
              </TableHeaderColumn>
              <TableHeaderColumn>
                Played
              </TableHeaderColumn>
              <TableHeaderColumn>
                Won
              </TableHeaderColumn>
              <TableHeaderColumn>
                Drawn
              </TableHeaderColumn>
              <TableHeaderColumn>
                Lost
              </TableHeaderColumn>
              <TableHeaderColumn>
                Goal
              </TableHeaderColumn>
              <TableHeaderColumn>
                Difference
              </TableHeaderColumn>
              <TableHeaderColumn>
                Points
              </TableHeaderColumn>
            </TableRow>
          </TableHeader>,
          <TableBody
            allRowsSelected={false}
            deselectOnClickaway={true}
            displayRowCheckbox={false}
            multiSelectable={false}
            preScanRows={true}
            selectable={true}
            style={Object {}}
          >
            <TableRow
              displayBorder={true}
              hoverable={false}
              hovered={false}
              selectable={true}
              striped={false}
            >
              <TableRowColumn
                hoverable={false}
              >
                1
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                sample
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                4
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                7
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                3
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                6
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                5
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                +1
              </TableRowColumn>
              <TableRowColumn
                hoverable={false}
              >
                8
              </TableRowColumn>
            </TableRow>
          </TableBody>,
        ],
        "fixedFooter": true,
        "fixedHeader": true,
        "height": "inherit",
        "multiSelectable": false,
        "selectable": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "adjustForCheckbox": false,
            "children": <TableRow
              displayBorder={true}
              hoverable={false}
              hovered={false}
              selectable={true}
              striped={false}
            >
              <TableHeaderColumn>
                Position
              </TableHeaderColumn>
              <TableHeaderColumn>
                Team name
              </TableHeaderColumn>
              <TableHeaderColumn>
                Played
              </TableHeaderColumn>
              <TableHeaderColumn>
                Won
              </TableHeaderColumn>
              <TableHeaderColumn>
                Drawn
              </TableHeaderColumn>
              <TableHeaderColumn>
                Lost
              </TableHeaderColumn>
              <TableHeaderColumn>
                Goal
              </TableHeaderColumn>
              <TableHeaderColumn>
                Difference
              </TableHeaderColumn>
              <TableHeaderColumn>
                Points
              </TableHeaderColumn>
            </TableRow>,
            "displaySelectAll": false,
            "enableSelectAll": true,
            "selectAllSelected": false,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <TableHeaderColumn>
                  Position
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Team name
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Played
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Won
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Drawn
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Lost
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Goal
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Difference
                </TableHeaderColumn>,
                <TableHeaderColumn>
                  Points
                </TableHeaderColumn>,
              ],
              "displayBorder": true,
              "hoverable": false,
              "hovered": false,
              "selectable": true,
              "striped": false,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Position",
                },
                "ref": null,
                "rendered": "Position",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Team name",
                },
                "ref": null,
                "rendered": "Team name",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Played",
                },
                "ref": null,
                "rendered": "Played",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Won",
                },
                "ref": null,
                "rendered": "Won",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Drawn",
                },
                "ref": null,
                "rendered": "Drawn",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Lost",
                },
                "ref": null,
                "rendered": "Lost",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Goal",
                },
                "ref": null,
                "rendered": "Goal",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Difference",
                },
                "ref": null,
                "rendered": "Difference",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Points",
                },
                "ref": null,
                "rendered": "Points",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "allRowsSelected": false,
            "children": Array [
              <TableRow
                displayBorder={true}
                hoverable={false}
                hovered={false}
                selectable={true}
                striped={false}
              >
                <TableRowColumn
                  hoverable={false}
                >
                  1
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  sample
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  4
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  7
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  3
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  6
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  5
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  +1
                </TableRowColumn>
                <TableRowColumn
                  hoverable={false}
                >
                  8
                </TableRowColumn>
              </TableRow>,
            ],
            "deselectOnClickaway": true,
            "displayRowCheckbox": false,
            "multiSelectable": false,
            "preScanRows": true,
            "selectable": true,
            "style": Object {},
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "class",
              "props": Object {
                "children": Array [
                  <TableRowColumn
                    hoverable={false}
                  >
                    1
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    sample
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    4
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    7
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    3
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    6
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    5
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    +1
                  </TableRowColumn>,
                  <TableRowColumn
                    hoverable={false}
                  >
                    8
                  </TableRowColumn>,
                ],
                "displayBorder": true,
                "hoverable": false,
                "hovered": false,
                "selectable": true,
                "striped": false,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 1,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 1,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "sample",
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": "sample",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 4,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 4,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 7,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 7,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 3,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 3,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 6,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 6,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 5,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 5,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "+1",
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": "+1",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": 8,
                    "hoverable": false,
                  },
                  "ref": null,
                  "rendered": 8,
                  "type": [Function],
                },
              ],
              "type": [Function],
            },
          ],
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
